src/main/java/com/exadel/discountwebapp/vendor/mapper/VendorMapper.java

```diff
@@ -56,8 +67,16 @@ private void configureModelMapper() {
         return new PropertyMap<>() {
             @Override
             protected void configure() {
-                skip().setLocation(null);
+                skip().setLocations(null);
             }
         };
     }
+
+    private List<Location> getLocationFromIds(List<Long> listIds) {
+        var locations = locationRepository.findLocationByIdIn(listIds);
+        if(locations.size()!= listIds.size()){
```

Comment:
Ideally better iterate and show with id's were not found. But it's more comment, than recommendation)

Reply:
nice point
error messages should be informative enough to be able to understand and investigate the cause of error

Reply:
fixed